src/ConstructiveSolidGeometry/VolumePrimitives/Ellipsoid.jl

```diff
@@ -20,11 +20,23 @@ function Geometry(::Type{T}, ::Type{Ellipsoid}, dict::AbstractDict, input_units:
     r = parse_r_of_primitive(T, dict, input_units.length)
     φ = nothing
     θ = nothing
-    e = Ellipsoid{T,ClosedPrimitive,typeof(r),typeof(φ),typeof(θ)}(
-        r = r, 
-        φ = φ, 
-        θ = θ
-    )
+    e = if r isa Tuple{T,T}
+        Ellipsoid{T,ClosedPrimitive,T,typeof(φ),typeof(θ)}(
+            r = r[2], 
+            φ = φ, 
+            θ = θ
+        ) - Ellipsoid{T,ClosedPrimitive,T,typeof(φ),typeof(θ)}(
```

Comment:
Shouldn't this be an `OpenPrimitive`? Or do we want the inner radius excluded?
Also, treating a `Sphere` with an inner radius `r1 != 0` results in failing tests.

Reply:
Ah correct, that should be an OpenPrimitive. 
Yes, the IO is not perfect yet. Also for other primitives. 
That is something I want to talk about but not tackle in this PR.